libvast/vast/aliases.hpp

```diff
@@ -61,5 +63,7 @@ constexpr id max_events = max_id + 1;
 /// The data type for an enumeration.
 using enumeration = uint32_t;
 
+using synopsis_options = std::map<std::string, data>;
```

Comment:
What was your rationale for picking `std::map` over `caf::config_value_map`? The frequently use the latter and its `get_or` accessor yields much nicer code than manually looking for a key and manually matching on the `data` inside.

Reply:
The only reason was that `data` as mapped type seemed to be more flexible and VAST-centric. But since we have `config_value_map` many times as an options carrier, I do find this type semantically more appropriate. The latest commit switches to it.